examples/social-network/README.md

```diff
@@ -0,0 +1,15 @@
+# Social Network Example
+
+This example was created as part of the [talk Fran Mendez did at QCon Plus 2021](https://plus.qconferences.com/plus2021/speakers/fran-mendez).
+
+## Run it
+
+There are three services. All of them can be started by running:
```

Comment:
Do you have by any chance a diagram of how the apps are interconnected? I think it is worth adding more context here.

Reply:
I didn't have it but I created it just for you right now 😘 :trollface:

Reply:
Thanks for the diagram, helped me understand what's going on.

Small one @fmvilas you have 

- server folder

Do you think it's worth naming that `websocket-server` to match the diagram?

Reply:
👍 Done!